### DISCUSSION TITLE
Github stops working in Jupyter Notebook

### DISCUSSION THREAD

**Discussion by lloydtripp** [community/community#102178] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi,

I'm having issues getting github copilot to consistently work in VS studio code with a jupyter notebook file. When I'm working on an old notebook, I do not receive any suggestions for even simple tasks (Asking to add 1 + 1). This will work on a new file but will eventually stop working as I continue editing the notebook. If I make a copy of a notebook , it'll work a couple times but eventually stop bringing up suggestions. One thing I'm noticing is that the model deployment ID becomes empty in Github's Output logs.

Before, when it works.
`[INFO] [streamChoices] [2024-01-27T02:26:29.512Z] solution 0 returned. finish reason: [stop]
[INFO] [streamChoices] [2024-01-27T02:26:29.514Z] solution 1 returned. finish reason: [stop]
[INFO] [streamChoices] [2024-01-27T02:26:29.514Z] request done: headerRequestId: [d605c66a-5f65-4ebd-8d15-06d78e3cefe7] model deployment ID: [wb5041db867a5]
[INFO] [default] [2024-01-27T02:26:33.394Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2024-01-27T02:26:33.529Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 135 ms
[INFO] [streamChoices] [2024-01-27T02:26:33.530Z] solution 2 returned. finish reason: [stop]
[INFO] [streamChoices] [2024-01-27T02:26:33.532Z] solution 1 returned. finish reason: [stop]
[INFO] [streamChoices] [2024-01-27T02:26:33.533Z] solution 0 returned. finish reason: [stop]
[INFO] [ghostText] [2024-01-27T02:26:33.533Z] GhostText later completion: [
    print(i)]
[INFO] [ghostText] [2024-01-27T02:26:33.533Z] GhostText later completion: [
    print(i)]
[INFO] [streamChoices] [2024-01-27T02:26:33.533Z] request done: headerRequestId: [5908a1a9-6e53-4550-8daa-78be94a044f3] model deployment ID: [wb5041db867a5]
[INFO] [ghostText] [2024-01-27T02:26:34.116Z] Found inline suggestions locally
[INFO] [ghostText] [2024-01-27T02:26:34.909Z] Found inline suggestions locally
[INFO] [default] [2024-01-27T02:26:35.670Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2024-01-27T02:26:35.778Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 107 ms
[INFO] [streamChoices] [2024-01-27T02:26:35.779Z] solution 0 returned. finish reason: [stop]
[INFO] [streamChoices] [2024-01-27T02:26:35.779Z] request done: headerRequestId: [a59f1797-66a6-4dd5-823a-3afd570b5aec] model deployment ID: [wb5041db867a5]`

After it stops working
`[INFO] [default] [2024-01-27T02:27:09.783Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2024-01-27T02:27:09.896Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 113 ms
[INFO] [streamChoices] [2024-01-27T02:27:09.897Z] request done: headerRequestId: [4cca211f-66aa-4921-b6a9-b5c8454e39e9] model deployment ID: []
[INFO] [ghostText] [2024-01-27T02:27:10.420Z] Cancelled during debounce
[INFO] [ghostText] [2024-01-27T02:27:10.525Z] Cancelled during debounce
[INFO] [default] [2024-01-27T02:27:10.653Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2024-01-27T02:27:10.788Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 134 ms
[INFO] [streamChoices] [2024-01-27T02:27:10.789Z] request done: headerRequestId: [e948b4be-0d11-48f0-b63a-8467778f2fbf] model deployment ID: []`

**Comment by lloydtripp** [comment]
For some reason, after the 11th line in this code block. The deployment ID is empty and suggestions aren't provided. AFter line 6, I'm able to get suggestions.

![image](https://github.com/community/community/assets/6070112/bf2939aa-af90-4045-8867-85e14c3bba6a)
![image](https://github.com/community/community/assets/6070112/a14b5fb8-6a54-4060-9ffa-6e6af97b522b)

**Comment by lloydtripp** [comment]
I've figured out the answer. Apparently, Github copilot thinks I'm using offensive language despite using the words in biological context. Another user reported a similar issue. See the two examples below. One shows Copilot code suggestion after removing the word "Homo" from "homo sapiens". 
![Screenshot 2024-01-28 174017](https://github.com/community/community/assets/6070112/92319a29-6dee-4c1f-9e0a-586b2cfa0371)
![Screenshot 2024-01-28 174100](https://github.com/community/community/assets/6070112/f7fc1099-4731-421f-a04c-fb0d87ed5904)

Even the model deployment ID changes dependent on the prior text. How do I get this fixed so I can progress with my scientific research?

Link to other user's reference to this issue: https://github.com/orgs/community/discussions/49002#discussioncomment-6736204

**Comment by raldone01** [comment]
Wow. It doesn't event tell you that it thinks the code is offensive. I just abbreviated homography with homo and that just made it stop working without any indication?
